server/db/models/campaigns.js

```diff
@@ -10,5 +11,76 @@ export default db.Model.extend({
   },
   calls() {
     return this.hasMany(Call);
+  },
+  responses() {
+    return this.hasMany(Response).through(Call);
+  },
+  getExport() {
+    // getExport should only be called on campaigns that were loaded with related calls.responses
+    const calls = this.relations.calls;
+
+    // some records have duplicate props, this renames the props and returns
+    // a version of the object that no longer has the duplicate prop name.
+    function samePropProcessing(object, objectType, propName, index) {
+      // create a copy because linter likes functional
+      const cloneObject = object;
```

Comment:
I might be misunderstanding but is this meant to be a true clone as in a new object with the same enumerable props as the input object?
Have you considered using a method to explicitly "clone" the object? const/let declarations reference their assignment. Which means that instead of mutating a new object created from the input, this will mutate the input object.

Reply:
Also, by 'functional' do you mean that the linter thought you were creating a class via functional inheritance?

Reply:
I'm going to use lodash.clone to get a shallow clone.